src/main/java/frc/team2412/robot/Main.java

```diff
@@ -7,17 +7,21 @@
 import edu.wpi.first.wpilibj.RobotBase;
 
 /**
- * Do NOT add any static variables to this class, or any initialization at all. Unless you know what
- * you are doing, do not modify this file except to change the parameter class to the startRobot
+ * Do NOT add any static variables to this class, or any initialization at all.
+ * Unless you know what
+ * you are doing, do not modify this file except to change the parameter class
+ * to the startRobot
```

Comment:
Please fix line breaks

Reply:
I must have hit enter here by accident or something, because running the formatter again removes the random linebreak